### ISSUE TITLE
[2.0.3] Database Errors - Laravel 5.1

### ISSUE THREAD

**Issue by nerijunior** [adrenth/redirect#19]
### 1- When I visited some page:
```
SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`lyx`.`adrenth_redirect_clients`, CONSTRAINT `redirect_client` FOREIGN KEY (`redirect_id`) REFERENCES `adrenth_redirect_redirects` (`id`) ON DELETE CASCADE) (SQL: insert into `adrenth_redirect_clients` (`redirect_id`, `timestamp`, `day`, `month`, `year`, `crawler`) values (1, 2016-07-11 09:52:31, 11, 7, 2016, ))
```

\* I haven't any redirect at the database

### 2 - When I tried to uninstall the plugin I got:

```
SQLSTATE[42000]: Syntax error or access violation: 1067 Invalid default value for 'created_at' (SQL: alter table `adrenth_redirect_redirects` drop `last_used_at`)" on line 662 of .../vendor/laravel/framework/src/Illuminate/Database/Connection.php
```

**Comment by adrenth** [comment]
Thanks for noticing. Will provide a fix this week.

**Comment by adrenth** [comment]
I fixed issue 1 in release 2.0.4.

For your second issue please see this thread on the OctoberCMS forums: http://octobercms.com/forum/post/installation-error-with-mysql

I think it has something to do with your MySQL version.

**Comment by nerijunior** [comment]
Thanks!
